cmd/fmt.go

```diff
@@ -209,5 +219,6 @@ func init() {
 	formatCommand.Flags().BoolVarP(&fmtParams.overwrite, "write", "w", false, "overwrite the original source file")
 	formatCommand.Flags().BoolVarP(&fmtParams.list, "list", "l", false, "list all files who would change when formatted")
 	formatCommand.Flags().BoolVarP(&fmtParams.diff, "diff", "d", false, "only display a diff of the changes")
+	formatCommand.Flags().BoolVarP(&fmtParams.errOnDiff, "err-on-diff", "e", false, "non zero exit code on reformat")
```

Comment:
Hmm, I think I would prefer to reuse the `--fail` flag from `opa eval` here. WDYT?

Reply:
👍 let's give others some time to chime in before changing though :)

Reply:
Opps didn't see this before I went ahead and changed it, though I think `fail` is better if it is used in our cmds.

Reply:
No worries, easily changed again if needed 😄 LGTM